topics/single-cell/tutorials/scrna-case_alevin/tutorial.md

```diff
@@ -14,6 +14,17 @@ redirect_from:
 questions:
   - I have some single cell FASTQ files I want to analyse. Where do I start?
 
+histories:
+<div class="dropdown">
+  <button class="btn btn-secondary dropdown-toggle" type="button" data-toggle="dropdown" aria-expanded="false">
+    {% icon galaxy-history %} Answer Key Histories
+  </button>
+  <div class="dropdown-menu">
+    <a class="dropdown-item" href="#">[galaxy.eu](https://humancellatlas.usegalaxy.eu/u/j.jakiela/h/generating-a-single-cell-matrix-using-alevin-3) </a>
+    <a class="dropdown-item" href="#"> [Older Alevin version](https://humancellatlas.usegalaxy.eu/u/wendi.bacon.training/h/cs1pre-processing-with-alevin---answer-key) </a>
+  </div>
+</div>
```

Comment:
```suggestion
histories:
  current:
    label: "UseGalaxy.eu"
    history: https://humancellatlas.usegalaxy.eu/u/j.jakiela/h/generating-a-single-cell-matrix-using-alevin-3
  older-alevin:
    label: "Older Alevin version"
    history: https://humancellatlas.usegalaxy.eu/u/wendi.bacon.training/h/cs1pre-processing-with-alevin---answer-key
```

Reply:
oh score! Actually, that's a good point - I think the histories index could be a separate level